packages/@lwc/engine/scripts/build.js

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+
+const path = require('path');
+const fs = require('fs-extra');
+
+const LWC_ENGINE_DIR = path.resolve(__dirname, '..');
+const LWC_ENGINE_DOM_DIR = path.resolve(__dirname, '../../engine-dom');
+
+fs.mkdirp(path.resolve(LWC_ENGINE_DIR, 'dist'));
+
+fs.copySync(
```

Comment:
Is it guaranteed that engine-dom has already been built at this point?

Reply:
Yes. More details here: https://github.com/salesforce/lwc/pull/1916#discussion_r438643036